ui/app/selectors/selectors.js

```diff
@@ -284,6 +288,11 @@ export function getIsMainnet(state) {
   return networkType === MAINNET;
 }
 
+export function getIsTestnet(state) {
+  const networkType = getCurrentChainId(state);
+  return TEST_CHAINS.includes(networkType);
```

Comment:
Nit: `networkType` is not the best variable name here. Elsewhere we use `networkType` or `type` to refer to internal string constants that we have for each built-in network (the "network identifier").

```suggestion
  const chainId = getCurrentChainId(state);
  return TEST_CHAINS.includes(chainId);
```